examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/BulkMetadataResponseTest.java

```diff
@@ -62,12 +62,24 @@ void apisIsNullWhenResponseIsNull() throws IOException {
 
     @Test
     void apisIsSerializedAsAnEmptyArrayWhenNotInitialised() throws IOException {
-      BulkMetadataResponse BulkMetadataResponse = new BulkMetadataResponse();
+      BulkMetadataResponse bulkMetadataResponse = new BulkMetadataResponse();
 
-      jsonContent = jacksonTester.write(BulkMetadataResponse);
+      jsonContent = jacksonTester.write(bulkMetadataResponse);
 
       assertThat(jsonContent).extractingJsonPathArrayValue("apis").isEmpty();
     }
+
+    @Test
+    void apiVersionIsSerialized() throws IOException {
+      BulkMetadataResponse bulkMetadataResponse = new BulkMetadataResponse();
```

Comment:
one for a follow-up, but I'd say that this is a bit too verbose, let's instead aim for:

```suggestion
      BulkMetadataResponse response = new BulkMetadataResponse();
```

(but also let's use `var`)

Reply:
will do!